BigANNBinaryEmbeddingOnlyScearioBase.cs

```diff
@@ -110,7 +110,7 @@ protected override ContainerProperties GetContainerSpec(string containerName)
             return properties;
         }
 
-        protected async Task PerformIngestion(IngestionOperationType ingestionOperationType, int startVectorId, int totalVectors)
+        protected async Task PerformIngestion(IngestionOperationType ingestionOperationType, int? startTagId, int startVectorId, int totalVectors)

```

Comment:
for replacing we also do a range replacing ?

Reply:
Yes, range defined by tags